diff --git a/libavfilter/Makefile b/libavfilter/Makefile
index 5123540..8a9fad8 100644
--- a/libavfilter/Makefile
+++ b/libavfilter/Makefile
@@ -466,6 +466,10 @@ OBJS-$(CONFIG_YAEPBLUR_FILTER)               += vf_yaepblur.o
 OBJS-$(CONFIG_ZMQ_FILTER)                    += f_zmq.o
 OBJS-$(CONFIG_ZOOMPAN_FILTER)                += vf_zoompan.o
 OBJS-$(CONFIG_ZSCALE_FILTER)                 += vf_zscale.o
+OBJS-$(CONFIG_PLUSGLSHADER_FILTER)           += vf_plusglshader.o
+OBJS-$(CONFIG_LUTGLSHADER_FILTER)            += vf_lutglshader.o
+OBJS-$(CONFIG_FADEGLSHADER_FILTER)           += vf_fadeglshader.o
+OBJS-$(CONFIG_PIPGLSHADER_FILTER)            += vf_pipglshader.o
 
 OBJS-$(CONFIG_ALLRGB_FILTER)                 += vsrc_testsrc.o
 OBJS-$(CONFIG_ALLYUV_FILTER)                 += vsrc_testsrc.o
diff --git a/libavfilter/allfilters.c b/libavfilter/allfilters.c
index 1183e40..ae2956c 100644
--- a/libavfilter/allfilters.c
+++ b/libavfilter/allfilters.c
@@ -444,6 +444,10 @@ extern AVFilter ff_vf_yaepblur;
 extern AVFilter ff_vf_zmq;
 extern AVFilter ff_vf_zoompan;
 extern AVFilter ff_vf_zscale;
+extern AVFilter ff_vf_plusglshader;
+extern AVFilter ff_vf_lutglshader;
+extern AVFilter ff_vf_fadeglshader;
+extern AVFilter ff_vf_pipglshader;
 
 extern AVFilter ff_vsrc_allrgb;
 extern AVFilter ff_vsrc_allyuv;
